### DISCUSSION TITLE
ARGOCD_CONTROLLER_REPLICAS equivalent in argocd-cmd-params-cm

### DISCUSSION THREAD

**Discussion by reggie-k** [argoproj/argo-cd#7599] [Q&A] [answered]
I am using ArgoCD 2.1.6 
What would be the key equivalent of ARGOCD_CONTROLLER_REPLICAS env var in the config map?

**Comment by pasha-codefresh** [comment, marked as answer]
Hi @reggie-k . According to controller manifest https://github.com/argoproj/argo-cd/blob/master/manifests/base/application-controller/argocd-application-controller-statefulset.yaml i can see that this env var cant be passed use config map
